provision/start-agent.sh

```diff
@@ -50,4 +50,4 @@ EOF
 chmod +x /etc/cron.daily/docker-cleanup
 
 ## Startup agent
-sudo systemctl enable buildkite-agent && sudo systemctl start buildkite-agent
+sudo systemctl enable buildkite-agent && sudo systemctl start buildkite-agent --tags "ubuntu,general-node"
```

Comment:
I think what you actually want is, prior to starting the service:
```
sudo mkdir -p /etc/systemd/system/buildkite-agent.service.d
echo '[Service]' | sudo tee -a /etc/systemd/system/buildkite-agent.service.d/override.conf
echo 'ExecStart=' | sudo tee -a /etc/systemd/system/buildkite-agent.service.d/override.conf
echo 'ExecStart=/usr/bin/buildkite-agent start --tags "ubuntu, general-node"' | sudo tee -a /etc/systemd/system/buildkite-agent.service.d/override.conf
```

Reply:
Thanks - yep, I think that would do it - I've stumbled across an easier way in https://buildkite.com/docs/agent/v3/configuration

In the default /etc/buildkite-agent/buildkite-agent.cfg, almost everything is commented out, including
```
# Tags for the agent (default is "queue=default")
# tags="key1=val2,key2=val2"
```

So I think we can (as sudo) append a tags line to here, and that should get picked up...